enterprise/server/backends/pubsub/pubsub.go

```diff
@@ -144,8 +147,15 @@ func (p *ListPubSub) Subscribe(ctx context.Context, channelName string) interfac
 	}
 }
 
-// To keep things simple for now, we maintain streams with a single value per element stored under the following key.
-const streamDataField = "data"
+const (
+	// To keep things simple for now, we maintain streams with a single value per element stored under the following key.
+	streamDataField = "data"
+	// Key prefix to identify monitored channels.
+	monitoredKeyPrefix = "monitoredPubSub/"
+	// For "monitored" channels, we publish a dummy message at channel creation time so we have a means to verify
+	// whether the object still exists in Redis.
```

Comment:
Is this mostly to be able to distinguish between streams that are "uninterrupted" (i.e. the Redis shard stayed up the whole time and the stream didn't get rehashed to a different shard) and streams that are "partial/broken" due to a rehash/node failure? If so, wdyt about calling this something like "streamStartMarkerMessage", since this is sort of a sentinel value to mark the start of the stream, and it will only be present if the stream is uninterrupted?

Reply:
Thanks for the suggestion, that's a good name.